scripts/aznfs_install.sh

```diff
@@ -103,9 +210,8 @@ else
     # Does not support CentOS for now.
 fi
 
-install_exit_code=$?
-if [ install_exit_code -ne 0 ]; then
-    echo "[FATAL] Error installing aznfs (exit code: $install_exit_code). See '$install_cmd' command logs for more information."
+if [ $install_error -ne 0 ]; then
+    eecho "[FATAL] Error installing aznfs (Error: $install_error). See '$install_cmd' command logs for more information."
 fi
 
-echo "Latest version of aznfs is installed."
+secho "Latest version of AZNFS is installed."
```

Comment:
Instead of saying latest version, say "Version $RELEASE_NUMBER of aznfs mount helper is successfully installed!"